### PR TITLE
Changelogs for Digital Twin Manager 1020.1.13

### PR DESCRIPTION
**Pull request by lra-sag**
This PR contains the changelogs for Digital Twin Manager 1020.1.13

### CODE REVIEW

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Fixed asset creation issue after importing sample asset models"
```
Comment (rahultsag) [inline content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md:3]: Proposing Title : "Imported sample asset models now display full hierarchy correctly"

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1509"
+version: "1020.1.13"
+---
+Previously, after importing the sample asset model, the hierarchy was not displayed correctly when creating an asset for the associated asset model. With this change, users can now view the complete hierarchy of the selected asset model.
```
Comment (rahultsag) [inline content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md:17]: Proposing this description "Previously, importing sample asset models sometimes resulted in an incomplete asset hierarchy during creation. This fix ensures the entire asset hierarchy is correctly displayed after importing sample asset models, enabling users to accurately create new assets."

**Review by rahultsag — DISMISSED** [review]

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md (hunk 2 above)
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md]: Modified it to : Previously, after importing a sample asset model, in **New asset** page, the asset hierarchy was incomplete when the sample asset model was selected. With this change, users can now view the entire hierarchy of the selected asset model.

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1509"
+version: "1020.1.13"
+---
+Previously, after importing a sample asset model, in **New asset** page, the asset hierarchy was incomplete when the sample asset model was selected. With this change, users can now view the entire hierarchy of the selected asset model.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md:17]:
```suggestion
Previously, after importing a sample asset model in the **New asset** page, the asset hierarchy was incomplete when the sample asset model was selected. With this change, users can now view the entire hierarchy of the selected asset model.
```

**Review by pavanshivajirao — APPROVED** [review]

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Fixed asset hierarchy issue after importing sample asset models in new asset page"
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md:3]:
```suggestion
title: "Asset hierarchy correctly shown in New asset page after importing a sample asset model
```

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md (hunk 1 above)
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md]: Modified it to: Asset hierarchy correctly shown in New asset page after importing a sample asset model

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md (hunk 2 above)
Comment (rahultsag) [reply content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md]: It's not a change but a fix. I would then suggest correcting this in the last line, "With this fix, users can now view the entire hierarchy of the selected asset model correctly."

File: content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
```
Comment (rahultsag) [inline content/change-logs/application-enablement/dtm-1020.1.13-fixed-asset-creation-issue-after-importing-sample-asset-models.md:2]: date seems to be missing

**Pull request merged**